test/models/line_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class LineTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
delete?

Reply:
oops 😬  I'll write some quick tests for the validations